draft-ietf-suit-report.md

```diff
@@ -458,7 +500,7 @@ A SUIT\_Component\_Capability is similar to a SUIT\_Component\_ID, with one diff
 
 When reporting capabilities, it is OPTIONAL to report capabilities that are declared mandatory by the SUIT Manifest {{I-D.ietf-suit-manifest}}. Capabilities defined by extensions MUST be reported.
 
-Additional capability reporting can be added as follows: if a manifest element does not exist in this map, it can be added by specifying the CBOR path to the manifest element in an array and using this as the key. For example SUIT\_Dependencies, as described in {{I-D.ietf-suit-trust-domains}} could have an extension added, which was key 3 in the SUIT\_Dependencies map. This capability would be reported as: \[3, 3, 1\] => \[3\], where the key consists of the key for SUIT\_Manifest (3), the key for SUIT\_Common (3), and the key for SUIT\_Dependencies (1). Then the value indicates that this manifest processor supports the extension (3).
+Additional capability reporting can be added as follows: if a manifest element does not exist in this map, it can be added by specifying the CBOR path to the manifest element in an array and using this as the key. For example SUIT\_Dependencies, as described in {{I-D.ietf-suit-trust-domains}}, Section 5.2.2, could have an extension added, which was key 3 in the SUIT\_Dependencies map. This capability would be reported as: \[3, 3, 1\] => \[3\], where the key consists of the key for SUIT\_Manifest (3), the key for SUIT\_Common (3), and the key for SUIT\_Dependencies (1). Then the value indicates that this manifest processor supports the extension (3).
```

Comment:
```suggestion
Additional capability reporting can be added as follows: if a manifest element does not exist in this map, it can be added by specifying the CBOR path to the manifest element in an array and using this as the key. For example SUIT\_Dependencies, as described in {{Section 5.2.2 of I-D.ietf-suit-trust-domains}}, could have an extension added, which was key 3 in the SUIT\_Dependencies map. This capability would be reported as: \[3, 3, 1\] => \[3\], where the key consists of the key for SUIT\_Manifest (3), the key for SUIT\_Common (3), and the key for SUIT\_Dependencies (1). Then the value indicates that this manifest processor supports the extension (3).
```